discord/ext/commands/bot.py

```diff
@@ -916,14 +971,18 @@ async def unload_extension(self, name: str, *, package: Optional[str] = None) ->
         await self._remove_module_references(lib.__name__)
         await self._call_module_finalizers(lib, name)
 
-    async def reload_extension(self, name: str, *, package: Optional[str] = None) -> None:
+    async def reload_extension(self, name: str, /, *, package: Optional[str] = None) -> None:
```

Comment:
I think this one is fine as a dual keyword/positional.

Reply:
```suggestion
    async def reload_extension(self, name: str, *, package: Optional[str] = None) -> None:
```